test/e2e/end_user_test.go

```diff
@@ -16,20 +14,6 @@ import (
 var _ = Describe("Openshift on Azure end user e2e tests [EndUser]", func() {
 	defer GinkgoRecover()
 
-	BeforeEach(func() {
-		namespace := nameGen.generate("e2e-test-")
```

Comment:
We run project creation before each test as opposed to before each suite to create a separate namespace for every test that runs for end users.

Reply:
+1 this is important, it allows each test to ensure it is working with a clean slate rather than trying to reason about existing state.  It also makes clean up easier since we can just blast the namespace, assuming that the test isn't creating cluster scoped objects

Reply:
Thanks for catching this, I had moved this to `BeforeSuite` but it does make more sense to have it on `BeforeEach`

Reply:
fixed in b196fa3

Reply:
Please revert this diff as it was before.